src/components/ChatEntry.js

```diff
@@ -1,22 +1,44 @@
 import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp.js';
+import { useState } from 'react';
+
+
+
 
 const ChatEntry = (props) => {
+  const [likeButton, setLikeButton] = useState('🤍') 
```

Comment:
As mentioned, since the like button setting gets set here to be the empty heart regardless of the actual value in `props`, if one of the messages _had_ already been liked, then it would still have the empty heart.

But like I said above in `App`, this doesn't really impact our case, but would have the same issues I outlined if it did.